x/ethbridge/keeper/msg_server.go

```diff
@@ -269,17 +269,17 @@ func (srv msgServer) UpdateCethReceiverAccount(goCtx context.Context,
 		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.CosmosSender)
 	}
 
-	err = srv.Keeper.ProcessUpdateCethReceiverAccount(ctx,
-		cosmosSender, cethReceiverAddress)
+	err = srv.Keeper.ProcessUpdateNativeTokenReceiverAccount(ctx,
+		cosmosSender, nativeTokenReceiverAddress)
 	if err != nil {
-		logger.Error("keeper failed to process update ceth receiver account.", errorMessageKey, err.Error())
+		logger.Error("keeper failed to process update native token receiver account.", errorMessageKey, err.Error())
 		return nil, err
 	}
 
-	logger.Info("sifnode emit update ceth receiver account event.",
+	logger.Info("sifnode emit update native token receiver account event.",
 		"CosmosSender", msg.CosmosSender,
```

Comment:
we can just do "msg", msg

Reply:
it is a easier way to print whole message, very useful, thanks